src/Data/Repositories/FUTXORepository.cs

```diff
@@ -50,6 +53,38 @@ public async Task<List<FMUTXO>> GetAll()
             throw new NotImplementedException();
         }
 
+        public async Task<List<FMUTXO>> GetByWalletId(int walletId)
+        {
+            await using var applicationDbContext = await _dbContextFactory.CreateDbContextAsync();
+            
+            return await applicationDbContext.FMUTXOs.Where(x => x.Wallet.Id == walletId)
+                .Include(x => x.Tags).ToListAsync();
+        }
+
+        public async Task<List<FMUTXO>> GetFromUTXOs(List<FMUTXO> utxos)
```

Comment:
this is a weird function, if you already have the list of utxos why do you need to get them from the db?

Reply:
You are right, I will rethink this. Initially, this was planned to be a method to retrieve a set of utxos from the database directly instead of filtering at the code level.